historyarchive/archive_pool.go

```diff
@@ -204,6 +204,16 @@ func (pa *ArchivePool) GetCheckpointHAS(chk uint32) (HistoryArchiveState, error)
 	})
 }
 
+func (pa *ArchivePool) GetLatestLedgerSequence() (uint32, error) {
+	has, err := pa.GetRootHAS()
+	if err != nil {
+		log.Error("Error getting root HAS from archive", err)
+		return 0, errors.Wrap(err, "failed to retrieve the latest ledger sequence from history archive")
+	}
+
+	return has.CurrentLedger, nil
+}
```

Comment:
I think this needs to round-robin like the other methods, right? So something like

```suggestion
	var state HistoryArchiveState
	return state, pa.runRoundRobin(func(ai ArchiveInterface) error {
		var err error
		state, err = ai.GetRootHAS()
		return err
	})
}
```

Reply:
Oh wait, you're delegating to the pool's own method anyway! Nevermind :)